docs/sources/auth/generic-oauth.md

```diff
@@ -237,3 +242,20 @@ Config:
 ```bash
 role_attribute_path = contains(info.groups[*], 'admin') && 'Admin' || contains(info.groups[*], 'editor') && 'Editor' || 'Viewer'
 ```
+
+
+### Groups mapping
+
+>  Only available in Grafana Enterprise v8.1+
+
+With Team Sync you can map your Generic OAuth groups to teams in Grafana so that your users will automatically be added to the correct teams.
```

Comment:
```suggestion
>  Available in Grafana Enterprise v8.1 and later versions.

With Team Sync you can map your Generic OAuth groups to teams in Grafana so that the users are automatically added to the correct teams.
```